init/cli/logging.go

```diff
@@ -57,8 +58,17 @@ func NewAlwaysAppending() FileFlags {
 }
 
 type FileLoggers struct {
-	flags FileFlags
-	mode  os.FileMode
+	flags   FileFlags
+	mode    os.FileMode
+	rotated map[string]bool
```

Comment:
nit: It's generally best practice to implement sets using `map[string]struct{}`, where existence in the map is sufficient.

Also I assume this logger is not used concurrently?

Reply:
Each returned logger is not used concurrently, but I am not sure if multiple loggers to the same path are used concurrently.